ddp-workspace/projects/ddp-rarex/src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+import { RoutePaths } from '../../router-resources';
+
+interface NavLink {
+  translationKey: string;
+  routePath: string;
+  external?: boolean;
+}
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+export class FooterComponent implements OnInit {
+  readonly rarexUrl = 'https://rare-x.org/';
```

Comment:
Is there anyway to make this configurable? Can we get this from `location.origin`? It will be nice if clicking `Home` will go back to whatever dev/test site we're on instead of going to the production site.

Reply:
It might be confusing at first, but this is a link to an external Rare-X website. Pepper version is going to hosted on a subpath of that domain.

Reply:
We will have to handle the `<base href="/" />` tag in `index.html` so that in production it says something like `<base href="/sub-path/" /> so that Angular routing works as expected.